libs/wallet-ui/src/components/transaction-details/transaction-details.tsx

```diff
@@ -35,13 +36,14 @@ type SectionListProps = {
 const compileSectionList = ({
   transaction,
   showStatus,
-  explorerUrl,
+  explorerLinks,
```

Comment:
suggestion (non-blocking): i might consider compiling the urls in the parent, and passing them in here as `txUrl, blockUrl` as strings, instead of an object of functions